### PR TITLE
[Wave] Use reciprocal to compute softcap logits

### PR DESCRIPTION
**Pull request by harsh-nod**
This PR uses reciprocals instead of divs to
compute the logits faster.

### CODE REVIEW

**Review by raikonenfnu — APPROVED** [review]

**Comment by raikonenfnu** [comment]
lgtm but not sure if we need to update lit

**Comment by raikonenfnu** [comment]
lgtm but not sure if we need to update lit

**Comment by harsh-nod** [comment]
> lgtm but not sure if we need to update lit

thanks, just updated it

**Review by raikonenfnu — APPROVED** [review]
